jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -2002,11 +1616,11 @@ ecma_builtin_typedarray_prototype_slice (ecma_value_t this_arg, /**< this argume
  *         Returned value must be freed with ecma_free_value.
 */
 static ecma_value_t
-ecma_builtin_typedarray_prototype_to_locale_string_helper (ecma_object_t *this_obj, /**< TypedArray object */
+ecma_builtin_typedarray_prototype_to_locale_string_helper (ecma_object_t *obj_p, /**< TypedArray object */
                                                            uint32_t index) /** array index */
 {
-  ecma_typedarray_type_t class_id = ecma_get_typedarray_id (this_obj);
-  lit_utf8_byte_t *typedarray_buffer_p = ecma_typedarray_get_buffer (this_obj);
+  ecma_typedarray_type_t class_id = ecma_get_typedarray_id (obj_p);
```

Comment:
Can't we pass a reference to the typedarray info struct and use these from there?